src/libs/actions/IOU.js

```diff
@@ -2117,6 +2117,30 @@ function payMoneyRequest(paymentType, chatReport, iouReport) {
     Navigation.dismissModal(chatReport.reportID);
 }
 
+function detachReceipt(transactionID, reportID) {
+    const transaction = allTransactions[`${ONYXKEYS.COLLECTION.TRANSACTION}${transactionID}`];
```

Comment:
Do we need an `|| {}` at the end in case we don't have the current transaction data in Onyx?